src/rvoice/fluid_rvoice.c

```diff
@@ -361,12 +361,17 @@ fluid_rvoice_write (fluid_rvoice_t* voice, fluid_real_t *dsp_buf)
     return count;
 
   /*************** resonant filter ******************/
+  
   fluid_iir_filter_calc(&voice->resonant_filter, voice->dsp.output_rate,
-  		        fluid_lfo_get_val(&voice->envlfo.modlfo) * voice->envlfo.modlfo_to_fc +
- 		        fluid_adsr_env_get_val(&voice->envlfo.modenv) * voice->envlfo.modenv_to_fc);
+                    fluid_lfo_get_val(&voice->envlfo.modlfo) * voice->envlfo.modlfo_to_fc +
+                    fluid_adsr_env_get_val(&voice->envlfo.modenv) * voice->envlfo.modenv_to_fc);
 
   fluid_iir_filter_apply(&voice->resonant_filter, dsp_buf, count);
-
+  
+  /* additonal high-pass filter - only uses the fixed modulator, no lfos... */
```

Comment:
"custom filter" may be better. And "the fixed modulator" is a bit misleading, it sounds like there is a default modulator for that custom filter in the source somewhere.